lib/client.js

```diff
@@ -914,6 +914,98 @@ BucketsMdapiClient.prototype.versionInternal = function _version(opts, cb) {
 };
 
 
+/**
+ * Gets a batch of garbage.
+ *
+ * @param {Object} opts              - request parameters
+ * @param {String} opts.request_id   - request uuid
+ * @param {Function} cb              - callback
+ */
+BucketsMdapiClient.prototype.getGarbageBatch =
+function getGarbageBatch(opts, cb) {
+    assert.object(opts, 'opts');
+    assert.uuid(opts.request_id, 'opts.request_id');
+    assert.func(cb, 'cb');
+
+    var rpcctx = this.ctxCreateForCallback(cb);
+    var callback = this.makeReleaseCb(rpcctx, cb);
+
+    if (rpcctx) {
+        var log = rpc.childLogger(rpcctx, {});
+
+        rpc.rpcCommonBufferData({
```

Comment:
Could switch to the recentish `rpc.rpcCommonSingleMessage` method which handles the `            if (garbageBatch.length !== 1) {` logic you have below.

Reply:
Sure. That didn't exist when this PR was created, but I'll switch to using that now.